docs/codeql/codeql-overview/codeql-changelog/codeql-cli-2.21.0.rst

```diff
@@ -207,5 +207,5 @@ JavaScript/TypeScript
 
     *   Intersection :code:`&&`
     *   Subtraction :code:`--`
-    *   :code:`\\q` quoted string
+    *   :code:`\q` quoted string
```

Comment:
I think we should skip the changes in this file, as it is undoing the escaping fixes we made manually.